src/mito2/src/sst/file.rs

```diff
@@ -145,18 +142,10 @@ pub struct FileMeta {
     pub level: Level,
     /// Size of the file.
     pub file_size: u64,
+    /// Maximum uncompressed row group size of the file. 0 means unknown.
+    pub max_row_group_uncompressed_size: u64,
```

Comment:
non-blocking: Is it possible to get this during compaction time so we don't need to add one more field to the FileMeta.